src/views/LoginPage/LoginPage.tsx

```diff
@@ -11,7 +11,18 @@ export interface QuoteI {
 
 export default function LoginPage() {
   const { t } = useTranslation();
-  const randomIndex = getRandomNum(0, 11);
+  const [quoteIndex, setQuoteIndex] = useState(getRandomNum(0, 11));
+
+  const prevIndex = (quoteIndex + 11) % 12;
+  const activeQuoteClass = quoteIndex === 0 ? styles.activeQuote : '';
+
+  useEffect(() => {
+    const intervalId = setInterval(() => {
+      const newIndex = getRandomNum(0, 11);
+      setQuoteIndex(newIndex);
+    }, 2000);
```

Comment:
Change to 5000 ms, because it looks not very good

Reply:
Yeh, the time was set for testing. Will change.